assets/src/edit-story/elements/video/output.js

```diff
@@ -48,6 +49,7 @@ function VideoOutput({ element, box }) {
     alt,
     layout: 'fill',
     loop: loop ? 'loop' : undefined,
+    noaudio: isMuted ? 'noaudio' : undefined,
```

Comment:
Since AMP validation came up, we could add a JS unit test here to ensure AMP validation if this attribute is present.